content/compose/reference/_index.md

```diff
@@ -105,7 +92,7 @@ webapp:
     - "/data"
 ```
 
-If the `docker-compose.admin.yml` also specifies this same service, any matching
+If the `compose.admin.yml` also specifies this same service, any matching
 fields override the previous file. New values, add to the `webapp` service
 configuration.
```

Comment:
Not related to this change, but this description is inaccurate. Some fields are merged, not overridden. Labels, for example:

```yaml
# compose.yaml
services:
  webapp:
    build:
      context: .
      labels: 
        com.example.foo: "foo"
```

```yaml
# overrides.yaml
services:
  webapp:
    build:
      context: .
      labels: 
        com.example.name: "Alice"
```

```console
$ docker compose -f compose.yaml -f overrides.yaml build
$ docker image inspect compose-webapp --format json | jq '.[].Config.Labels'
{
  "com.docker.compose.project": "compose",
  "com.docker.compose.service": "webapp",
  "com.docker.compose.version": "2.23.0",
  "com.example.foo": "foo",
  "com.example.name": "Alice"
}
```

`buildx bake` has similar behavior and I've described how duplicate fields either merge with or override previous definitions in the bake file reference, see here: https://docs.docker.com/build/bake/reference/#file-format